sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -112,6 +112,11 @@ class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContex
       checkFilterPredicate('_1 === true, classOf[Eq[_]], true)
       checkFilterPredicate('_1 <=> true, classOf[Eq[_]], true)
       checkFilterPredicate('_1 !== true, classOf[NotEq[_]], false)
+
+      checkFilterPredicate(('_1 in(true)).asInstanceOf[Predicate],
```

Comment:
Nit: `('_1 in(true))` looks a little bit weird, maybe `'_1.in(true)`? This also applies to all the `in` predicates below.